libnm-core/nm-setting-connection.c

```diff
@@ -1982,4 +2005,34 @@ nm_setting_connection_class_init (NMSettingConnectionClass *setting_class)
 		                   G_PARAM_CONSTRUCT |
 		                   NM_SETTING_PARAM_FUZZY_IGNORE |
 		                   G_PARAM_STATIC_STRINGS));
+	/**
+	 * NMSettingConnection:mdns:
+	 *
```

Comment:
If you build with --enable-gtk-doc, you'll see that this comment ends up in `./clients/common/settings-docs.c` file (this also breaks `make check`).

You need to generated file over to `./clients/common/settings-docs.c.in`, and commit the changes as well.